### ISSUE TITLE
Bitcoin Core 0.21: Not able to connect 

### ISSUE THREAD

**Issue by tiero** [bwt-dev/bwt-electrum-plugin#1]
I'm running latest `bwt` on my linux with a Bitcoin Core 0.21 pruned.

I keep receiving this warning when I click on Save & Connect. I'm waiting but cannot connect to my instance.

With latest version of Bitcoin Core all was flawless, will try to downgrade and will see 

```
WARN bwt::electrum » rpc #6 server.banner failed: JSON-RPC error: JSON decode error: missing field `addnode`
```

**Comment by shesek** [comment]
Thanks for the report!

This is due to [`rust-bitcoincore-rpc`](https://github.com/rust-bitcoin/rust-bitcoincore-rpc) being incompatible with v0.21. I [sent a PR](https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/158) to address this, which is currently pending review.

I also [implemented a workaround](https://github.com/bwt-dev/bwt/commit/a6878c8a5dc64b5ff5e2b4555ea651e4528b4a12) in bwt that doesn't require the changes in `rust-bitcoincore-rpc`, but this didn't get released yet.

For now, you can set the following in your `bitcoin.conf` file to re-enable the deprecated RPC fields:

```
deprecatedrpc=getpeerinfo_addnode
deprecatedrpc=banscore
deprecatedrpc=whitelisted
```

**Comment by shesek** [comment]
I opened an issue on the main `bwt` tracker for visibility: https://github.com/bwt-dev/bwt/issues/78

**Comment by shesek** [comment]
The fix was released in v0.2.2.
